classes/startup.rb

```diff
@@ -0,0 +1,43 @@
+class Startup
+  def options_list
+    puts ''
+    puts "Please choose an option by selecting a number:
+    1. List all books
+    2. List all musics
+    3. List of games
+    4. Add a book
+    5. Add a music
+    6. Add a game
+    7. Exit"
+    gets.chomp
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
@ZahraArshia you have done great on switching the case. In my opinion we have to try to solve linter error instead of disable

Reply:
Thank you @bushmusi to check it, I'm open to suggestions about it

Reply:
@ZahraArshia now our startup console is ready. It is time to merge it